ooniprobe/AppDelegate.mm

```diff
@@ -23,10 +23,14 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     #ifdef RELEASE
     CrashlyticsKit.delegate = self;
     [Fabric with:@[[Crashlytics class]]];
+    [[NotificationService sharedNotificationService] registerProbe];
```

Comment:
So, this basically means that, whenever the app is loaded, we unconditionally register probe?

Reply:
No, it will call the registerprobe function, that will exit if the probe is already registered. 
All my doubts are in the registerProbe function